src/Shared/Shared/Configurations/Environment.cs

```diff
@@ -0,0 +1,37 @@
+namespace EShop.Shared.Configurations;
+
+/// <summary>
+/// Provides access to environment-specific configuration values used in the application.
+/// </summary>
+public static class AppEnvironment
+{
+    /// <summary>
+    /// Retrieves postgresql database configuration values from environment variables.
+    /// </summary>
+    /// <remarks>
+    /// This method is intended for development environments where connection details
+    /// are provided via environment variables (e.g., from a .env file or local shell).
+    ///
+    /// Expected environment variables:
+    /// - POSTGRES_PORT
+    /// - POSTGRES_DB
+    /// - POSTGRES_USER
+    /// - POSTGRES_PASSWORD
+    /// </remarks>
```

Comment:
Not sure about this implementation in terms of security, I should probably use AzureVault, or something else. But this is necessary for development purposes only.

In dev mode, environment variables will be loaded from `.env`, but in prod they will come from system envs.